engineyard/ey-cookbooks-stable-v5
# [FB-2260] Update Percona-Server Versions

Description of your patch
-------------
Updates the default/latest Percona-Server versions for `5.6` to `5.6.46.86.2`, and of `5.7` to `5.7.28.31`

Recommended Release Notes
-------------
Updates the default/latest Percona-Server versions for `5.6` to `5.6.46.86.2`, and of `5.7` to `5.7.28.31`

Estimated risk
-------------
Low

Components involved
-------------
`mysql` cookbook

Description of testing done
-------------
1. Boot a new environment with the labrea stack release, built from the changes made for the mysql cookbook
2. Check for the latest available mysql version available for MySQL version 5.6 and 5.7
3. Verify if the MySQL version matches to `5.6.46.86.2` in 5.6 and `5.7.28.31` in 5.7

QA Instructions
-------------

- Check the default versions by booting a solo environment for MySQL v5.5, v5.6 and v5.7
- Check the default versions for a multi-environment for MySQL versions v5.5, v5.6 and v5.7
